exposures/info/block- squid-analysis-reports.yaml

```diff
@@ -0,0 +1,20 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-squid-reports-block
+  namespace: default # use your own labels here
+spec:
+  message: Access deined for  file squid-reports
+  tags : ["sarg","squid-reports","exposure"]
+  selector:
+    matchLabels:
+       app: any  # use your own labels here
+  file:
+    matchPatterns: 
+     - pattern: /**/squid-reports
```

Comment:
Can you attach ref that confirms the confidence for keeping this path
I've tried to look into these 2 but didn't find useful info:
- https://github.com/projectdiscovery/nuclei-templates/blob/master/exposures/logs/squid-analysis-report-generator.yaml
- https://github.com/projectdiscovery/nuclei-templates/blob/master/technologies/waf-detect.yaml

Reply:
@raviknox There is a very high chance applying this policy can break the application

Reply:
https://www.tecmint.com/sarg-squid-analysis-report-generator-and-internet-bandwidth-monitoring-tool/#:~:text=Step%204%3A%20Assessing%20Sarg%20Report - Ref. doc for the used paths of the files